integrations/huggingface-transformers/tutorials/sentiment-analysis/sentiment-analysis-cli.md

```diff
@@ -0,0 +1,328 @@
+# Sentiment Analysis: Sparse Transfer Learning with the CLI
+
+In this example, you will sparse transfer learn a 90% pruned BERT model onto some sentiment-analysis datasets using SparseML's CLI.
+
+### **Sparse Transfer Learning Overview**
+
+Sparse Transfer Learning is very similiar to the typical transfer learing process used to train NLP models, where we fine-tune a pretrained checkpoint onto a smaller downstream dataset. With Sparse Transfer Learning, we simply start the fine-tuning process from a pre-sparsified checkpoint and maintain sparsity while the training process occurs.
+
+### Pre-Sparsified BERT
+
+SparseZoo, Neural Magic's open source repository of pre-sparsified models, contains a 90% pruned version of BERT, which has been sparsified on the upstream Wikipedia and BookCorpus datasets with the masked language modeling objective. [Check out the model card](https://sparsezoo.neuralmagic.com/models/nlp%2Fmasked_language_modeling%2Fobert-base%2Fpytorch%2Fhuggingface%2Fwikipedia_bookcorpus%2Fpruned90-none). We will use this model as the starting point for the transfer learning process.
+
+**Let's dive in!**
+
+## Installation
+
+Install SparseML via `pip`.
+
+```bash
+pip install sparseml[torch]
+```
+
+## Sparse Transfer Learning onto SST2 (GLUE Task)
+
+SparseML's CLI enables you to kick-off sparsification workflows with various utilities like creating training pipelines, dataset loading, checkpoint saving, metric reporting, and logging handled for you. 
+
+All we have to do is pass a couple of key arguments: 
+- `--model_name_or_path` specifies the starting checkpoint to load for training
+- `--task` specifies a glue dataset to train with 
```

Comment:
The command below contains `task_name` instead. We should use `task` as defined here or make it clear that they are the same